### PR TITLE
Mondoo Client v5.20.0

### PR DESCRIPTION
**Pull request by benr**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/mondoo-cli.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  url "https://releases.mondoo.io/mondoo/#{version}/mondoo_#{version}_darwin_universal.pkg"
+  name "Mondoo"
+  desc "Mondoo Client CLI for the Mondoo Policy as Code Platform"
+  homepage "https://mondoo.io/"
+
+  livecheck do
+    url "https://releases.mondoo.io/mondoo/latest"
+    strategy :page_match
+    regex(%r{href='\.\./(\d+.\d+.\d+)}i)
```
Comment (neutric) [inline Casks/mondoo-cli.rb:13]: Thanks for your contribution to `homebrew-cask`, @benr!

`strategy :page_match` should only be made explicitly when needed. It is the default except for special cases such as GitHub-URLs.
```
regex(/href=.*?v?(\d+(?:.\d+)+)/i)
```

Comment (benr) [reply Casks/mondoo-cli.rb:12]: Strategy removed.

Comment (neutric) [reply Casks/mondoo-cli.rb:12]: Thanks! Can you please also update to the more flexible, "go-to" regex?

File: Casks/mondoo-cli.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "52932b4c2d5e8355e745edb96de20811e0c42a9c61338d17b4cab3da95fc0399"
+
+  url "https://releases.mondoo.io/mondoo/#{version}/mondoo_#{version}_darwin_universal.pkg"
+  name "Mondoo"
+  desc "Mondoo Client CLI for the Mondoo Policy as Code Platform"
+  homepage "https://mondoo.io/"
+
+  livecheck do
+    url "https://releases.mondoo.io/mondoo/latest"
+    regex(%r{href='\.\./(\d+.\d+.\d+)}i)
```
Comment (miccal) [inline Casks/mondoo-cli.rb:12]:
```suggestion
    url "https://releases.mondoo.io/mondoo/latest/index.html"
    regex(%r{href='\.\./(\d+(?:\.\d+)+)}i)
```

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @benr.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

File: Casks/mondoo-cli.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  uninstall script: {
+    executable: "/Library/Mondoo/#{version}/uninstall.sh",
+    sudo:       true,
+  }
+
+  zap trash: [
+                '/Library/Mondoo',
+                '/etc/opt/mondoo',
+                '/usr/local/bin/mondoo',
```
Comment (neutric) [inline Casks/mondoo-cli.rb:25]: Isn't this one taken care of by `uninstall.sh`?

Comment (miccal) [reply Casks/mondoo-cli.rb:25]: Also, these should be part of the `uninstall` stanza -- the `zap` stanza is for user-generated files.

Please read the documentation I linked above to add a `zap`.

**Comment by benr** [comment]
Closing PR.  We'll resubmit in the future.

**Pull request closed without merging**